api/deployments/models/deployment_builder.go

```diff
@@ -226,7 +245,7 @@ func (b *deploymentBuilder) buildDeploySummaryPipelineJobInfo() DeploymentSummar
 	if b.pipelineJob != nil {
 		jobInfo.CommitID = b.pipelineJob.Spec.Build.CommitID
 		jobInfo.PipelineJobType = string(b.pipelineJob.Spec.PipeLineType)
-		jobInfo.BuiltFromBranch = b.pipelineJob.Spec.Build.Branch
+		jobInfo.BuiltFromBranch = b.pipelineJob.Spec.Build.Branch //nolint:staticcheck
```

Comment:
Should this be using GitRef instead? (I just noticed there are no references to the new field her, but not sure if its on purpose or not :) 

```suggestion
		jobInfo.BuiltFromBranch = b.pipelineJob.Spec.Build..GetGitRefOrDefault()
```

Reply:
Your fix is incorrect - the DeploymentSummary (now DeploymentSummaryPipelineJobInfo) and Deployment have `gitRef` and `gitRefType` fields
Fixed